documentation/staging/content/managing-domains/accessing-the-domain/weblogic-admin-console.md

```diff
@@ -0,0 +1,228 @@
+---
+title: "Use the Administration Console"
+date: 2019-02-23T17:39:15-05:00
+draft: false
+weight: 1
+description: "Use the WebLogic Administration Console with domains running in Kubernetes."
+---
+
+{{< table_of_contents >}}
+
+### Introduction
+You can access the WebLogic Server Administration Console external to the Kubernetes cluster using the following approaches:
+
+   * [Use a load balancer](#use-a-load-balancer)
+
+   * Use an [Administration Server `NodePort`](#use-an-administration-server-nodeport).
+
+   * [Use a `kubectl port-forward` connection](#use-a-kubectl-port-forward-connection).
+
+**Notes:**
+ * For production use cases, Oracle recommends using a load balancer with ingress path routing rules and an SSL port to access the WebLogic Server Administration Console. 
+
+ * To verify that your load balancer, NodePort, or kubectl port-forward setup is working as expected, see [Test]({{< relref "#test" >}}).
+
+ * Do not use the WebLogic Server Administration Console to start or stop servers, or for scaling clusters. See [Starting and stopping servers]({{< relref "/managing-domains/domain-lifecycle/startup#starting-and-stopping-servers" >}}) and [Scaling]({{< relref "/managing-domains/domain-lifecycle/scaling.md" >}}). In addition, if your domain home type is either [Domain in Image]({{< relref "/samples/domains/domain-home-in-image/_index.md" >}}) or [Model in Image]({{< relref "/samples/domains/model-in-image/_index.md" >}}), then do not use the Administration Console to make changes to the WebLogic domain configuration as these changes are ephemeral and will be lost when servers restart. See [Choose a domain home source type]({{< relref "/managing-domains/choosing-a-model/_index.md" >}}).
```

Comment:
Minor: "In addition, ... " can now be turned into its own bullet (plus remove 'In addition').

Reply:
Changed.